### PR TITLE
modify gc7219 datasheet's reference link, fix #1785

### LINKED ISSUE #1785: [Bug Report]: gc7219 datasheet参考链接需要修改
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:
gc7219 datasheet参考链接需要修改

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by ethanlcz**

### CODE REVIEW

**Review by yong171966 — COMMENTED** [review]
looks good to me, approved

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 4e6e8b68c5900614af02dfb56298cc972a11e2f9

**Comment by skylarCai** [comment]
auto build fail,number:194

**Comment by skylarCai** [comment]
autotest fail,number:1533

File: haas_lib_bundles/python/libraries/gc7219/README.md [hunk 1]
```diff
@@ -182,5 +182,5 @@ MCU 与从设备传输数据时，首先将 CS 片选信号置为低电平，随
 </div>
```
Comment (skylarCai) [inline haas_lib_bundles/python/libraries/gc7219/README.md:182]: LGFM

**Review by skylarCai — COMMENTED** [review]
resolve security risk

**Pull request merged**